core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -7029,9 +7039,10 @@ class KafkaApisTest extends Logging {
       requestChannelRequest.context,
       consumerGroupHeartbeatRequest
     )).thenReturn(future)
-    kafkaApis = createKafkaApis(overrideProperties = Map(
-      GroupCoordinatorConfig.NEW_GROUP_COORDINATOR_ENABLE_CONFIG -> "true"
-    ))
+    kafkaApis = createKafkaApis(
+      overrideProperties = Map(GroupCoordinatorConfig.GROUP_COORDINATOR_REBALANCE_PROTOCOLS_CONFIG -> "classic,consumer"),
+      raftSupport = true
```

Comment:
why do we need the raft support now?

Reply:
because we use the kraft metadata cache now. i am not really happy with how `createKafkaApis` works but i refrained myself from changing everything in this patch :).

Reply:
So the default is to use ZK Metadata cache (fow now) 😅